test/tests/geo/searches.json

```diff
@@ -0,0 +1,157 @@
+[
+	{
+    "comment": "breweries near the couchbase office",
+		"search": {
+			"from": 0,
+			"size": 10,
+      "query": {
+        "location": {
+          "lon": -122.107799,
+          "lat": 37.399285
+        },
+        "distance": "100mi",
+				"field":"geo"
+      },
+      "sort": [
+       {
+         "by": "geo_distance",
+         "field": "geo",
+         "unit": "mi",
+         "location":{
+          "lon":-122.107799,
+          "lat":37.399285
+         }
+       }
+     ]
+		},
```

Comment:
the indenting of this JSON file looks off when viewed in github's PR reviewer screens?